templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} new car model</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" value="Submit">
+    </form>
+{% endblock %}

```

Comment:
Ensure that you have used 2 whitespaces for indentations in this `.html` file, as required by the checklist.